hicommonwealth/commonwealth
# Show skeleton loader on view snapshot and multiple snapshots page

## Link to Issue
Closes: implements task list item 2 and 3 from https://github.com/hicommonwealth/commonwealth/issues/4546
> 2- Multiple Snapshot Spaces page: if a community has multiple snapshot spaces connected, then there is a selection page that appears.
3- View Snapshot Proposal Page: [live example](https://commonwealth.im/dydx/snapshot/dydxgov.eth/0x0ea7f312f0970e0a9e4e34bd38af59dd79babcd11169282971df1fc210228359).

## Description of Changes
- Adds a skeleton loader to view snapshot page
- Adds a skeleton loader to multiple snapshots page

## "How We Fixed It"
- By adding params to show the skeleton loader
- By adding the markup for skeleton on the multiple snapshots page

## Test Plan
- Visit a snapshot proposal page and verify that it shows a skeleton loader when the content is loading
- Visit the multiple snapshots page: `/:chain/multiple-snapshots` and verify it shows a skeleton

## Deployment Plan
N/A

## Other Considerations
N/A
